### DISCUSSION TITLE
Dire Maul West (RDF) should no longer require Crescent Key

### DISCUSSION THREAD

**Discussion by andovcr** [chromiecraft/chromiecraft#6104] [Suggestions]
When an RDF group is placed into Dire Maul West (DM:F), the group cannot progress past Tendris Warpwood into Immol'thar's chamber, because the Crescent Key is required to open the Door in the Court of the Highborne.  The Crescent Key is only obtainable from a quest in Dire Maul East (DM:E).  Unless someone from the group has obtained the Crescent Key, is a Rogue that has leveled lockpicking (Lockpicking 300), is an engineer with Powerful Seaforium Charge (Engineering 275), or a blacksmith with Arcanite Skeleton Key (Blacksmithing 275), then the dungeon cannot be completed.

I propose that the door be automatically opened for RDF groups.

This is extremely similar to the request for BRD - Upper City: [Suggestion 5502](https://github.com/chromiecraft/chromiecraft/discussions/5502), (which was Accepted and changed on CC) but even worse for the player experience, because in BRD the means of entry to Upper City is available (though it takes time), whereas in Dire Maul the group is simply stuck.  This change would not speed up the time to complete the dungeon, or break any mechanics (that I'm aware of) due to the group still needing to kill Immol'thar (and thus clear the 5 Arcane Pylons), in order for Prince Tortheldrin to be attackable.  All this would do is make the content accessible for RDF groups, who are just trying to level.

**Comment by heyitsbench** [comment]
@Gultask 👀

**Comment by balleny** [comment]
some corrections:

> The Crescent Key is only obtainable from a quest in Dire Maul East (DM:E)
its a drop in DM:E

> is a Rogue that has leveled lockpicking (Lockpicking 300), is an engineer with Powerful Seaforium Charge (Engineering 275), or a blacksmith with Arcanite Skeleton Key (Blacksmithing 275)
Cresecent key doors act as 225 (at least on chromie right now), hence a lot cheaper / easier obtainable solutions work, too

Finally, this sounds more like an bug and therefore should be in bug reports. Or did you try to start an discussion about it in the first place?

**Reply by andovcr** [reply]
Good corrections - thanks Lys.

As far as I know, this is Blizz-like behavior (much like the BRD door prior to the Lyceum requiring a key on RDF), but that is just an assumption, not a proven fact.  So I thought starting this as a suggestion would be better, because I'm trying to match functionality to a different suggestion (so that players can complete the dungeon when put it via RDF).
